bin/build.sh

```diff
@@ -14,6 +14,8 @@
 
 set -euo pipefail
 
+yarn clean-dev:js
```

Comment:
Can we leave a comment to why we do it? (I don't think it's intuitive that we do it to, in theory, make the uberjar build faster as it may seem we may actually slow it down by doing a "clean build")